ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
   - return_fan_run_status_2
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+EXTOR:
+  description: "shade control with extension monitoring based on outside illuminance"
+  is_abstract: true
+  uses:
+  - shade_extent_percentage_command
+  - outside_illuminance_sensor
+  - low_limit_illuminance_setpoint
+  - high_limit_illuminance_setpoint
+
+DPHC2:
+  description: "Two-pipe heating and cooling control for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - supply_water_temperature_setpoint
+  uses:
+  - supply_water_temperature_sensor
+  - return_water_temperature_sensor
+  - water_riser_mode
+  - cooling_output_percentage_command
```

Comment:
are these valves, or merely control loop outputs? if they are valves, i would expect them to be called valves. same for heating below.

Reply:
Indeed, for the 2-Pipe RP we can directly use the valve because in this case the value represents the valves position and the heating or cooling load at once. (see explenation below)